### PR TITLE
Default terminal directory path resolution

### PR DESCRIPTION
**Pull request by kazelone**
Implemented a more robust and flexible way of specifying the cwd of default terminals, since I had issue with those in a multi-root workspace configuration.

I also added a webpack config to bundle the  extension in a smaller package.

Do wathever you want with this.

### CODE REVIEW

**Comment by BenWildeman** [comment]
could you split this down into different PR's, one for webpack, one for the path resolution

File: package.json [hunk 1]
```diff
@@ -80,63 +80,29 @@
         "title": "Tabulous: Reload default terminals"
       },
       {
-        "command": "tabulous.showTerminal1",
```
Comment (BenWildeman) [inline package.json]: deleting these are a breaking change

Comment (kazelone) [reply package.json]: True, didn't want to commit that. Was experiencing with using a single command with argument instead but didn't turn out to play well with keybindings so I dropped the idea.

**Comment by kazelone** [comment]
> 
> 
> could you split this down into different PR's, one for webpack, one for the path resolution

Yup, I can do that. Should have time to do it today.

**Pull request closed without merging**